package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "bundle": "webpack --progress --colors --mode=production",
     "build": "webpack --progress --colors --mode=development",
-    "test": "npm run bundle && karma start"
+    "test": "npm run build && karma start"
   },
   "author": "bahmni@thoughtworks.com",
   "license": "Apache-2.0",
```

Comment:
AGPLv3

Reply:
Sure will change it. I copied it from `BahmniApps` only. Should it be changed there too?

https://github.com/Bahmni/openmrs-module-bahmniapps/blob/master/ui/package.json

Reply:
Yes. We should. I will do that.

Reply:
Ignore, my prev comment. should be MPLv2.